next-app/src/utils/hooks/useColorMode.ts

```diff
@@ -0,0 +1,29 @@
+import { useState, useEffect } from 'react'
+
+import { COLOR_MODE } from 'constants/colorMode'
+
+export const useColorMode = () => {
+  const [isDarkMode, setIsDarkMode] = useState(false)
+
+  useEffect(() => {
+    const preferredColorMode = localStorage.getItem(`${COLOR_MODE}`)
+    const systemColorMode = window.matchMedia('(prefers-color-scheme: dark)')
+      .matches
+      ? 'dark'
+      : 'light'
+    const colorMode = preferredColorMode ?? systemColorMode
+    setIsDarkMode(colorMode === 'dark')
+  }, [])
+
+  const ToggleColorMode = () => {
```

Comment:
```suggestion
  const toggleColorMode = () => {
```
대문자로 시작하는 건 컴포넌트 이름이었으면 좋겠어요

Reply:
앗! 나의 실수